tests/test_functons.py

```diff
@@ -112,3 +118,80 @@ def test_distance_from_zero(test_distance_from_zero_input, test_distance_from_ze
 ])
 def test_square(distance_from_zero_type_input, distance_from_zero_type_expected):
```

Comment:
@nicolas13sochi could you fix this?
`test_square()` is defined on line 35 and 119 - one of them is redundant - please compare test cases from both and make sure to delete the one that is redundant.